### DISCUSSION TITLE
Error bad argument

### DISCUSSION THREAD

**Discussion by Marcoscopico** [rspamd/rspamd#4402] [Q&A]
Hello, good morning, I hope you can help me with the following error, I have already searched the configuration file, but I cannot find the error, I don't know if it is a variable that I have wrong, I would appreciate your help, thank you.

error:
init of /usr/share/rspamd/spamassassin.lua failed: /usr/share/rspamd/spamassassin.lua:1641: bad argument #1 to 'glob' (string expected, got boolean); trace: [1]:{[C]:-1 - glob [C]}; [2]:{/usr/share/rspamd/spamassassin.lua:1641 - [main]};

RSPAMD 2.1

**Comment by vstakhov** [comment]
The actual version of Rspamd is 3.4. Everything else is unsupported.
